lib/portlayer/exec/container_cache.go

```diff
@@ -27,32 +34,55 @@ import (
 type containerCache struct {
 	m sync.RWMutex
 
-	// cache maps containerVM to container ID && infrastructure ID
+	// cache by container id
```

Comment:
The single map has moids or UIDs -- so typically 2 entries for each containerVM -- one keyed off of moid another via UID.

Originally I had this as two maps -- that might come back to fruition in a near term change.